src/action.js

```diff
@@ -57,12 +58,14 @@ module.exports = async function run({
       return logWarning('PR contains non dependabot commits, skipping.')
     }
 
-    try {
-      await verifyCommits(commits)
-    } catch {
-      return logWarning(
-        'PR contains invalid dependabot commit signatures, skipping.'
-      )
+    if (!SKIP_COMMIT_VERIFICATION) {
+      try {
+        await verifyCommits(commits)
```

Comment:
Not related to your changes, but I don't see any valid reason to use `await` here. The `verifyCommits` function is not returning a `Promise`.